t/t.go

```diff
@@ -1050,10 +1052,31 @@ func validateAllowed(testSuite []string) {
 	}
 }
 
+func validateDownload() {
+	// 	if len(*download) == 0 {
+	// 		return
+	// 	}
+	// 	if *download != "ldbc" {
+	// 		log.Fatalf("Only ldbc is supported for download")
+	// 	}
+	if testSuiteContains("ldbc") || testSuiteContains("all") {
+		out, err := downloadLDBCFiles()
+		length := len(out)
+		for i := 0; i < length; i++ {
+			str := string(out[i])
+			log.Println("Error Downloading a file: ", str)
```

Comment:
just use small letters in panic comment, same below

Reply:
Resolved